interp/module.go

```diff
@@ -132,8 +132,7 @@ func DefaultOpen(ctx Ctxt, path string, flag int, perm os.FileMode) (io.ReadWrit
 
 func OpenDevImpls(next ModuleOpen) ModuleOpen {
 	return func(ctx Ctxt, path string, flag int, perm os.FileMode) (io.ReadWriteCloser, error) {
-		switch ctx.UnixPath(path) {
-		case "/dev/null":
+		if ctx.UnixPath(path) == "/dev/null" {
```

Comment:
please leave this as is - it's likely we're going to add others here, like `/dev/urandom` and `/dev/zero`.